llvm/lib/CodeGen/AtomicExpandPass.cpp

```diff
@@ -1803,6 +1805,8 @@ static ArrayRef<RTLIB::Libcall> GetRMWLibcall(AtomicRMWInst::BinOp Op) {
   case AtomicRMWInst::FSub:
   case AtomicRMWInst::UIncWrap:
   case AtomicRMWInst::UDecWrap:
+  case AtomicRMWInst::USubCond:
+  case AtomicRMWInst::USubSat:
     // No atomic libcalls are available for max/min/umax/umin.
```

Comment:
Comment was already out of date, but it probably shouldn't bother listing out all the names of the operations